File: TransCoder_tokenized_test_set_functions/cpp/DIFFERENCE_BETWEEN_HIGHEST_AND_LEAST_FREQUENCIES_IN_AN_ARRAY_1.cpp
int findDiff ( int arr [ ], int n ) {
  unordered_map < int, int > hm;
  for ( int i = 0;
  i < n;
  i ++ ) hm [ arr [ i ] ] ++;
  int max_count = 0, min_count = n;
  for ( auto x : hm ) {
    max_count = max ( max_count, x . second );
    min_count = min ( min_count, x . second );
  }
  return ( max_count - min_count );
}

File: TransCoder_tokenized_test_set_functions/python/DIFFERENCE_BETWEEN_HIGHEST_AND_LEAST_FREQUENCIES_IN_AN_ARRAY_1.py
def findDiff ( arr, n ) :
    mp = defaultdict ( lambda : 0 )
    for i in range ( n ) :
        mp [ arr [ i ] ] += 1
    max_count = 0;
    min_count = n
    for key, values in mp . items ( ) :
        max_count = max ( max_count, values )
        min_count = min ( min_count, values )
    return max_count - min_count
